### DISCUSSION TITLE
No preview for uploaded video

### DISCUSSION THREAD

**Discussion by wpas81** [filamentphp/filament#5516] [Help] [answered]
Hi

I've a form which uploads a video but when I navigate to the view or edit pages the video's not shown. It never gets past the 'waiting for size'-status 

Any advice on how to resolve this?

![image](https://user-images.githubusercontent.com/64780854/213981625-d8feec0e-5bc2-42c0-949b-81872a67b6ed.png)

```
return $form
	->schema([
		Card::make()
			->schema([
				Hidden::make('mime'),

				TextInput::make('title')
					->required()
					->maxLength(255)
					->autofocus(),
				
				FileUpload::make('path')
					->required()
					->acceptedFileTypes(['video/mp4','video/ogg','video/webm'])
					->maxSize(51200)
					->disk('videos')
					->visibility('public')
					->afterStateUpdated(fn (callable $set, $state) => $set('mime', $state?->getMimeType())),
			])
	]);
```

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/tricks/file-upload-previews-not-loading

**Reply by wpas81** [reply]
you save me again... I checked the console, the disk was correctly linked... but I forgot to add the 's' in httpS in the app_url.... 
So next time I'll also search the trics :)

**Comment by gcazin** [comment]
In case anyone comes across this problem again, simply add ->fetchFileInformation(false) to the FIleUpload component 😉

```php
Forms\Components\FileUpload::make('path')
     ->fetchFileInformation(false)
     ->directory('medias')
    ->required(),
```

**Reply by aliteleb** [reply]
Thank you bro ❤️❤️
